### DISCUSSION TITLE
Controller name giving an error

### DISCUSSION THREAD

**Discussion by tshad** [react-hook-form/react-hook-form#3632] [Q&A]
Using Bill Luo's demo on validation I have an issue the Controller and the name. I get the following error: 

_lastName is missing in the 'defaultValue' prop of either its Controller (https://react-hook-form.com/api#Controller) or useForm (https://react-hook-form.com/api#useForm)_

I get the same for firstName.

I think it is set up like the demo:

      <View>
        <Text style={styles.label}>First Name</Text>
        <Controller 
          name="firstName" 
          control={control} 
          rules={{required: "First name is required."}}
          render={props => (
            <TextInput 
              {...props} 
              style={styles.input}
              onChangeText={(value) => {
                (props.onChange(value))
              }}
            />
          )}
        />
      </View>
      <View>
        <Text style={styles.label}>Last Name</Text>
        <Controller 
          name="lastName" 
          control={control} 
          rules={{required: "Last name is required."}}
          render={props => (
            <TextInput 
              {...props} 
              style={styles.input}
              onChangeText={ (value) => {
                (props.onChange(value)) 
              }}
            />
        )}
        />
      </View>

Thanks.

**Comment by bluebill1049** [comment]
you are missing `defaultValue` at Controller or at `useForm`

**Comment by tshad** [comment]
Hi Bill, 

Thanks for the response. 

I saw that in the docs, but I can't see where it is set in the demo. Was it set somewhere else? 

![image](https://user-images.githubusercontent.com/12076957/101653446-32156b00-39f4-11eb-9ddc-fe214bc20ef8.png)


Thanks, 

Tom

**Comment by bluebill1049** [comment]
<img width="943" alt="Screen Shot 2020-12-10 at 8 56 51 am" src="https://user-images.githubusercontent.com/10513364/101693394-a704a580-3ac5-11eb-8332-2bb156d648f5.png">

https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r

**Comment by tshad** [comment]
Not sure I understand. I think I have what your demo had but not sure what I am missing that caused the error. I am missing a default value?

![image](https://user-images.githubusercontent.com/12076957/101703269-62cac400-3a37-11eb-87ac-bbed3b62e76e.png)
